bouncer/tests/full_bouncer.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe } from 'vitest';
+import { testBtcUtxoConsolidation } from './btc_utxo_consolidation';
+import { testDeltaBasedIngress } from './delta_based_ingress';
+import { testDoubleDeposit } from './double_deposit';
+import { testRotatesThroughBtcSwap } from './rotates_through_btc_swap';
+import { testRotateAndSwap } from './rotation_barrier';
+import { testSolanaVaultSettingsGovernance } from './solana_vault_settings_governance';
+import { serialTest } from '../shared/utils/vitest';
+
+// Tests that are run by the ci-main-merge one at a time
+describe('SlowTests', () => {
```

Comment:
It's not necessarily that they're slow - but that they can't be run in parallel, perhaps SerialTests is better

```suggestion
describe('SlowTests', () => {
```

Reply:
Already have a group called `SerialTests` in the fast bouncer file, so I didn't want to call it that. Maybe `ExtraSerialTests`?